src/app/models/projectNotification.ts

```diff
@@ -0,0 +1,36 @@
+import { CommentPeriod } from "./commentperiod";
```

Comment:
got a lint error on this line:

/Users/mwardle/Documents/bcgov/eagle-public/src/app/models/projectNotification.ts:1:31
ERROR: 1:31  quotemark  " should be '